extensions/roc-package-web-app-react/src/app/server/reactRenderer.js

```diff
@@ -28,8 +29,11 @@ const whiteListed = () => (
 const appConfig = whiteListed();
 
 export function initRenderPage({ script, css }, distMode, devMode, Header) {
-    const templatePath = rocConfig.runtime.template.path || `${myPath}/views`;
-    nunjucks.configure(getAbsolutePath(templatePath), {
+    const templatePaths = [defaultTemplatePath]
+        .concat(rocConfig.runtime.template.path || [])
```

Comment:
Should we change the order here, with the user specified directory coming before the default one? That would mean that we could keep the template name while still allowing this behavior.